Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1736,48 +1737,44 @@ namespace ScriptCanvasEditor
     
     void MainWindow::OnSaveCallBack(const VersionExplorer::FileSaveResult& result)
     {
-        const bool saveSuccess = result.fileSaveError.empty();
-        auto completeDescription = CompleteDescription(m_fileSaver->GetSource());
-        auto memoryAsset = completeDescription ? *completeDescription : m_fileSaver->GetSource();
-        int saveTabIndex = m_tabBar->FindTab(memoryAsset);
+        const bool saveSuccess = result.IsSuccess();
+
+        int saveTabIndex = -1;
```

Comment:
Fixes a bug in which a newly created file would save, but still have its screen display frozen.